test/start-kubernetes.sh

```diff
@@ -254,15 +255,17 @@ EOF
     pids=""
     for ip in ${IPS}; do
         SECONDS=0
-        NO_PROXY+=",$ip"
         #Wait for the ssh connectivity in the vms
         echo "Waiting for ssh connectivity on vm with ip $ip"
         while ! ssh $SSH_ARGS ${CLOUD_USER}@${ip} exit 2>/dev/null; do
             if [ "$SECONDS" -gt "$SSH_TIMEOUT" ]; then
                 die "timeout accessing ${ip} through ssh"
             fi
         done
+    done
```

Comment:
It's not obvious why the loop is split here. Please add a comment explaining it, otherwise we might end up reverting this again in the future.

Reply:
Good point,  thanks! Comment added and pushed (no functional changes)